corehq/apps/data_interfaces/static/data_interfaces/js/auto_update_rules.js

```diff
@@ -0,0 +1,133 @@
+hqDefine("data_interfaces/js/auto_update_rules", [
+    'jquery',
+    'knockout',
+    'underscore',
+    'hqwebapp/js/initial_page_data',
+    'analytix/js/google',
+    'hqwebapp/js/components.ko', // for pagination
+], function (
+    $,
+    ko,
+    _,
+    initialPageData,
+    googleAnalytics
+) {
+
+    var RuleViewModel = function (data) {
+        var self = {};
+        self.action_error = null;
+        self.active = data.active;
+        self.caseType = data.case_type;
+        self.editURL = data.edit_url;
+        self.id = data.id;
+        self.lastRun = data.last_run;
+        self.name = data.name;
+        self.status = data.status;
+        return self;
+    };
+
+    var RuleListViewModel = function (rules) {
+        var self = {};
+        self.rules = ko.observableArray(_.map(rules, RuleViewModel));
+        self.paginatedRules = ko.observableArray([]);
+        self.rulesById = ko.computed(function () {
+            return _.indexBy(self.rules(), 'id');
+        });
+
+        // pagination
+        self.itemsPerPage = ko.observable(5);
+        self.totalItems = ko.computed(function () {
+            return self.rules().length;
+        });
+        self.currentPage = 1;
+
+        self.goToPage = function (page) {
+            self.currentPage = page;
+            self.paginatedRules.removeAll();
+            var skip = (self.currentPage - 1) * self.itemsPerPage();
+            self.paginatedRules(self.rules().slice(skip, skip + self.itemsPerPage()));
+        };
+
+        self.onPaginationLoad = function () {
+            self.goToPage(1);
+        };
+
+        // actions
+        self.deleteRule = function (rule) {
+            $.ajax({
+                url: "",
+                type: "POST",
+                dataType: 'json',
```

Comment:
According to jquery docs, this is the default
> dataType (default: Intelligent Guess (xml, json, script, or html))
So sounds like it would probably work if not specified, but I figure it is easy enough to keep it?